hw/ip/prim/dv/prim_prince/crypto_dpi_prince/prince_ref.h

```diff
@@ -0,0 +1,342 @@
+/*
+Copyright 2016 Sebastien Riou
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
You can combine the license headers to:

```
// Copyright lowRISC contributors.
// Copyright 2016 Sebastien Riou
// Licensed under the Apache License, Version 2.0, see LICENSE for details.
// SPDX-License-Identifier: Apache-2.0
```

@asb please confirm. I did check the original repo, there is *no* NOTICE file we need to take care of.

Reply:
just a friendly ping to @asb - could you confirm whether @imphil's suggestion would work?

Reply:
Sorry for missing this one Udi - it looks good to me.

Reply:
Thanks @asb, @imphi!
Should we grant this file an exception to the license-check portion of CI in `util/licence-checker.hjson`?
Otherwise CI will keep failing due to the mismatched license header.

Reply:
Yes please.